src/System.Console/tests/TermInfo.cs

```diff
@@ -0,0 +1,132 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+using System.Runtime.InteropServices;
+using System.Text;
+using Xunit;
+
+public class TermInfo
+{
+    [Fact]
+    [PlatformSpecific(PlatformID.AnyUnix)]
+    public void VerifyInstalledTermInfosParse()
+    {
+        string[] locations = GetFieldValueOnObject<string[]>("_terminfoLocations", null, Type.GetType("ConsolePal.TermInfo.Database"));
```

Comment:
I'm surprised this works; I'd have thought a) you'd need to specify the full type name, including "System.", and b) that Type.GetType would only be looking at the current assembly and the core platform assembly, but not System.Console.dll.  I'd have expected you'd instead have needed something like:
```C#
typeof(Console).GetTypeInfo().Assembly.GetType("System.ConsolePal.TermInfo.Database")
```